### PR TITLE
Support html / css assets

### PR DESCRIPTION
**Pull request by vkalpias**

### CODE REVIEW

File: src/resources/resources_css.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    };
+
+    CssHandler.prototype = {
+        load: function (url, callback) {
+            pc.net.http.get(url, function (response) {
+                callback(null, response);
+            }, {
+                error: function (status, xhr, e) {
+                    callback(pc.string.format("Error loading html resource: {0} [{1}]", url, status));
```
Comment (daredevildave) [inline src/resources/resources_css.js]: "html"

File: src/resources/resources_html.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        open: function (url, data) {
+            return data;
+        },
+
+        patch: function (asset, assets) {
+        }
+    };
+
+    var createHtml = function (htmlString) {
+        return document.createRange().createContextualFragment(htmlString);
```
Comment (daredevildave) [inline src/resources/resources_html.js]: what does this return? what is the browser support for this?

Comment (Maksims) [inline src/resources/resources_html.js]: Seems like well supported: https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment
It takes valid html string, and converts it to abstract document fragment, which then can be appended as child to DOM.

Comment (vkalpias) [inline src/resources/resources_html.js]: Returns a DocumentFragment. Seems to be supported by all browsers except Safari and IE mobile.. I guess I can use a more universal method.

**Pull request merged**